doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/adding-data-sets-proc.adoc

```diff
@@ -0,0 +1,29 @@
+[id='adding-data-sets-proc_{context}']
+= Adding data sets
+
+You can create a data set to fetch data from an external data source and use that data for the reporting components.
+
+.Procedure
+. In {CENTRAL}, go to *Admin* -> *Data Sets*.
++
+The *Data Set Explorer* page opens.
+. Click *New Data Set* and select one of the following provider types:
++
+* *Bean:* Use to generate a data set from a Java class
+* *CSV:* Use to generate a data set from a remote or local CSV file
+* *SQL:* Use to generate a data set from an ANSI-SQL compliant database
+* *Elastic Search:* Use to generate a data set from Elastic Search nodes
+* *Prometheus:* Used to generate a data set using the Prometheus query
+* *Execution Server:* Use to generate a data set using the custom query feature of an Execution Server
++
+
+NOTE: You must configure *{KIE_SERVER}* for this option.
```

Comment:
"NOTE: You must configure *{KIE_SERVER}* for this option."

Can you clarify which "option".

Reply:
I have updated the content. It is for the Execution Server option.